### DISCUSSION TITLE
Roblox style OOP (nvm it kinda already existed my bad)

### DISCUSSION THREAD

**Discussion by Tormentous** [godotengine/godot-proposals#10171] [3D]
I wanna just do .parent.parent instead of having some line of code that has me trying to find something i cant access normally though $, its so annoying

**Comment by Meorge** [comment]
Wouldn't this be done with `get_parent().get_parent()` etc?

**Comment by theraot** [comment]
This is `get_parent().get_parent()` with `$` syntax, by the way: `$"../.."`.

**Comment by AThousandShips** [comment]
Parent shouldn't be a property as it's read-only, there's no `set_parent` because it's more complicated than that, so having it be a property would be confusing IMO, inexperienced users would try to assign to it and it'd be confusing that it won't work
